Problema com os comandos de setup no README

clonei o repo, e fiz o seguinte:
```
$ make install
virtualenv .venv
make: virtualenv: Command not found
Makefile:146: recipe for target 'virtualenv' failed
make: *** [virtualenv] Error 127
```
em seguida criei um virtual env com o comando:
```
$ python -m venv .venv
```
e rodei novamente o make install:
```
$ make install
virtualenv .venv
make: virtualenv: Command not found
Makefile:146: recipe for target 'virtualenv' failed
make: *** [virtualenv] Error 127
```
Lendo o make file descobri o pq: O Makefile usa o pacote virtualenv para criar o ambiente se você não possui ele instalado o comando falha. Fiz `pip install virtualenv` e rodei novamente o `make install`. Então sim o `make up` funciona e serve o site como esperado no `localhost:8000`

--- a/README.md
+++ b/README.md
@@ -2,28 +2,41 @@ Site Pyladies Brasil
 ====================
 [![Build Status](https://app.codeship.com/projects/bca2dab0-d874-0134-15a2-326e4d300ce2/status?branch=master)](https://app.codeship.com/projects/bca2dab0-d874-0134-15a2-326e4d300ce2/status?branch=master)
 
-Instalando e Rodando
---------------------
-
-- Assumindo que seu git e virtualenv já estão configurados, faça o clone do repositório
-
-		$ git clone git@github.com:pyladies-brazil/br-pyladies-pelican.git
-
-- Instale os requisitos de python via pip com o comando:
-> Obs.: Esse comando criará uma virtual env, então rode-o fora de qualquer ambiente virtual.
-
-		$ make install
-
-- Rode o projeto
-
-		$ make up
-
-Abra o browser em [localhost:8000](http://localhost:8000) para ver o conteúdo gerado.
-
 Contribuindo
 ------------
 
-Para contribuir com o projeto veja o guia de [Contribuição](https://github.com/pyladies-brazil/br-pyladies-pelican/blob/master/CONTRIBUTING.md).
+Para contribuir com o projeto veja o guia de [Contribuição](https://github.com/pyladies-brazil/br-pyladies-pelican/blob/master/CONTRIBUTING.md). Lá você encontrará instruções detalhadas de como fazer a sua contribuição.
+
+Instalando e Rodando
+--------------------
+
+- Para apenas rodar localmente o site, você precisa do [virtualenv](https://virtualenv.pypa.io/en/stable/) instalado na sua máquina.
+
+Para instalar o virtualenv basta fazer:
+
+```console
+$ pip install virtualenv
+```
+- Assumindo que seu git e virtualenv já estão configurados, faça o clone do repositório
+
+``` console
+$ git clone git@github.com:pyladies-brazil/br-pyladies-pelican.git
+```
+
+- Rode o comando para criação de ambiente virtual e instalação das dependências
+
+``` console
+$ make install
+```
+> Obs.: Esse comando criará uma virtual env, então rode-o fora de qualquer ambiente virtual.
+
+- Rode o projeto
+
+``` console
+$ make up
+```
+
+Abra o browser em [localhost:8000](http://localhost:8000) para ver o conteúdo gerado.
 
 Links Úteis
 -----------

